test/unit/test_extension_wrapper_common.py

```diff
@@ -71,6 +73,30 @@ def test_bucketfs_credentials_default(mock_connect, filled_secrets):
     ))
 
 
+@unittest.mock.patch("pyexasol.connect")
+def test_bucketfs_credentials_internal(mock_connect, filled_secrets):
+
+    path_in_bucket = 'location'
+    internal_host = 'localhost'
```

Comment:
I guess, that we handle this as the default, will then be done in the UI

Reply:
These parameters would be set by the UI. The test imitates this.